client/my-sites/stats/controller.jsx

```diff
@@ -189,7 +189,7 @@ export function overview( context, next ) {
 		return (
 			context.params.period === filter.period ||
 			context.path.indexOf( filter.path ) >= 0 ||
-			( filter.altPaths && context.path.indexOf( filter.altPaths ) >= 0 )
+			( filter.altPaths && filter.altPaths.includes( context.path ) )
```

Comment:
Changes to the controller are unrelated to this PR, so let's omit them from here and propose them in a separate PR.